specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -11927,6 +11969,37 @@
           "$ref": "#/definitions/AzureAvsNodeType",
           "description": "AVS node type."
         },
+        "nodeTypes": {
+          "type": "array",
+          "description": "AVS node types.",
+          "items": {
+            "$ref": "#/definitions/AzureAvsNodeType"
+          }
+        },
+        "externalStorageTypes": {
+          "type": "array",
+          "description": "AVS external storage types.",
+          "items": {
+            "$ref": "#/definitions/ExternalStorageType"
+          }
+        },
+        "avsAssessmentScenario": {
+          "$ref": "#/definitions/AvsAssessmentScenario",
+          "description": "AVS Assessment Scenario."
+        },
+        "isVcfByolEnabled": {
+          "type": "boolean",
```

Comment:
Can boolean be modelled as string to be extensible in future?

Reply:
In our scenario, we will be providing a checkbox on the UI to the customer which will tell us whether customer has applied VCF BYOL or not. So, I think boolean will make more sense in our scenario.